### DISCUSSION TITLE
Apply same visualization settings to several files

### DISCUSSION THREAD

**Discussion by jmtc7** [IENT/YUView#579] [Q&A]
Hi! First of all, thanks you so much for your great work developing this app! However, I use it to visualize and compare different versions of the same YUV image, so I spend quite a lot of time setting the same configuration (width, height, YUV format, and UV interleaved) each time I open a new file, or even if I open 5 or 6 at the same time. Is there any way of setting the same parameters for all the files in the playlist? If not, I believe it would be a great feature able to accelerate many pipelines!

Thanks again for your work and support

**Comment by ChristianFeldmann** [comment]
Hi! Not sure if this is a solution for you but YUView can obtain the exact YUV settings (width/height/subsampling/bitDepth...) from the filename. So if the filename is set in a correct way, then the files should be recognised correctly.
